README.md

```diff
@@ -138,12 +138,13 @@ az sapmonitor provider-instance create \
-|----------------------------------|----------------------------------------------------------------------------------------------------------------------------------| ----------------------------------------------------- |
-| SapHana                          | {"hanaHostname":"10.0.0.6","hanaDbName":"SYSTEMDB","hanaDbSqlPort":30013,"hanaDbUsername":"SYSTEM"," hanaDbPassword":"password"} | None                                                  |
-| PrometheusHaCluster              | {"prometheusUrl":"http://10.0.0.20:9664/metrics"}                                                                                | {"sid":"HA1","hostname":"hdb1-0","clustername":"HA1"} |
-| PrometheusOS                     | {"prometheusUrl":"http://10.0.0.21:9100/metrics"}                                                                                | None                                                  |
-| MsSqlServer                      | {"sqlHostname":"10.0.0.6","sqlPort":1433,"sqlUsername":"sqladmin","sqlPassword":"password"}                                      | {"sid":"HA1"}                                         |
+| Provider Type                    | Provider Properties                                                                                                                          | Required Metadata                                     |
+|----------------------------------|----------------------------------------------------------------------------------------------------------------------------------            | ----------------------------------------------------- |
+| SapHana                          | {"hanaHostname":"10.0.0.6","hanaDbName":"SYSTEMDB","hanaDbSqlPort":30013,"hanaDbUsername":"SYSTEM"," hanaDbPassword":"password"}             | None                                                  |
+| PrometheusHaCluster              | {"prometheusUrl":"http://10.0.0.20:9664/metrics"}                                                                                            | {"sid":"HA1","hostname":"hdb1-0","clustername":"HA1"} |
+| PrometheusOS                     | {"prometheusUrl":"http://10.0.0.21:9100/metrics"}                                                                                            | None                                                  |
+| MsSqlServer                      | {"sqlHostname":"10.0.0.6","sqlPort":1433,"sqlUsername":"sqladmin","sqlPassword":"password"}                                                  | {"sid":"HA1"}                                         |
+| SapNetWeaver                     | {"sapHostName":"10.0.0.6","sapInstanceNr":"20"}<br>{"sapHostName":"netweaverapp","sapInstanceNr":"20","sapSubdomain":"fabrikam.contoso.com"} | {"sapSid":"HA1"}                                      |
```

Comment:
Have we added sapSubdomain already? I don't see it in the other repo

Reply:
It's being tested and hasn't been merged yet.
I can circle back when that is done.